datadog_checks_base/tests/conftest.py

```diff
@@ -53,3 +54,23 @@ def kerberos():
             conditions=[CheckDockerLogs(compose_file, "ReadyToConnect")],
         ):
             yield common_config
+
+
+@pytest.fixture(scope="session")
+def uds_path():
+    if Platform.is_mac():
+        # See: https://github.com/docker/for-mac/issues/483
+        pytest.skip('Sharing Unix sockets is not supported by Docker for Mac.')
```

Comment:
I explored the alternative of running an in-process server, such as Gunicorn. But Gunicorn in particular is a huge pain to setup programmatically (I couldn't make it work), since it was designed for CLI use. I know [Uvicorn](https://www.uvicorn.org) is flexible enough to be used in this start/stop fashion with little effort, but it's Py3+ only. Apart from Gunicorn I'm not sure which WSGI server that supports UDS we could use. So I went with skipping integration tests on Mac for now…